include/revng/Lift/BinaryFile.h

```diff
@@ -449,12 +449,14 @@ class BinaryFile {
   using LabelList = llvm::SmallVector<Label *, 6u>;
 
   using LabelIntervalMap = interval_map<MetaAddress, LabelList, compareAddress>;
+  using BinaryHandleT = llvm::object::OwningBinary<llvm::object::Binary>;
```

Comment:
We don't really use trailing `T`s for type names.
It's not great, but if this gives you issues with field names, use `TheBinaryHandle`.